packages/core/src/components/MenuButton/MenuButton.module.scss

```diff
@@ -73,6 +73,7 @@
   border-color: var(--disabled-background-color);
   color: var(--disabled-text-color);
   cursor: not-allowed;
+  pointer-events: none;
```

Comment:
I think it is a bit risky
what if we would just not apply the css for focus and hover in disabled mode?
```
.wrapper:focus:not(.disabled),
.wrapper:hover:not(.disabled) {
  background-color: var(--primary-background-hover-color);
}
```